### ISSUE TITLE
canShareVia invokes the success callback when it should not - iOS 11.2

### ISSUE THREAD

**Issue by rajkumar-osm** [EddyVerbruggen/SocialSharing-PhoneGap-Plugin#937]
**Device information**
iPhone 5s
iOS 11.2
Facebook app is not installed.

window.plugins.socialsharing.canShareVia('com.apple.social.facebook', 'msg', null, null, null, 
function(success) {
   do some stuff....
}, function(error) {
   alert(error); 
});

**Issue description**
When the Facebook app is not installed, canShareVia method should invoke the error callback, which is working perfectly with my iPhone5s running iOS 10.

When I test it on iPhone5s running iOS 11.2, it is invoking the success callback in both the cases where Facebook app is installed and Not installed.

**Comment by EddyVerbruggen** [comment]
Yep, since iOS11 this changed and I had to [always return `true`]( https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/blob/819361c16ea919a79795d3c85af276ddce0883da/src/ios/SocialSharing.m#L263). If you know of a good way to detect the app please share it.

**Comment by AbijeetP** [comment]
Created the pull request that fixes #937 
https://github.com/EddyVerbruggen/SocialSharing-PhoneGap-Plugin/pull/938
